packages/compass-indexes/src/modules/create-index/index.js

```diff
@@ -151,12 +147,19 @@ export const createIndex = () => {
     const state = getState();
     const spec = {};
 
-    // check for errors
+    // Check for field errors.
     if (state.fields.some((field) => field.name === '' || field.type === '')) {
       dispatch(handleError('You must select a field name and type'));
       return;
     }
 
+    // Check for collaction errors.
+    const collation = queryParser.isCollationValid(state.collationString);
+    if (state.isCustomCollation && collation === false) {
```

Comment:
Seems like the return types here are pretty messed up and we should fix that upstream, but probably would make sense to handle this here too for the time being. Return values that indicate that collation is not valid are not just `false`, but falsy it seems:

![image](https://user-images.githubusercontent.com/5036933/181743297-8473ea3e-bbe1-42de-828b-119f40f8268c.png)

So something like that should probably cover it a bit better:

```suggestion
    if (state.isCustomCollation && !collation) {
```